src/server/protocol.d.ts

```diff
@@ -1216,4 +1216,38 @@ declare namespace ts.server.protocol {
     export interface NavBarResponse extends Response {
         body?: NavigationBarItem[];
     }
+
+    export interface TsdOptions {

```

Comment:
These tsd interfaces will be deprecated shortly - is there a way to just add them to the file they are being consumed in so that you don't need to export them?